v2/cn/components.md

```diff
@@ -8,11 +8,13 @@
     @changes       30
 --->
 
-下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有的组件可以被激活或禁止。
-大部组件默认是激活的，也有部件不是。
-[LuaJIT](luajit.html)、 [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html) 默认是没有激活的。您需要通过以下选项在编译
-[OpenResty](openresty.html)的时候将它们各自激活， `--with-luajit`、
-`--with-http_drizzle_module`、 `--with-http_postgres_module`和 `--with-http_iconv_module` 。
+下面列表中的组件被用于构建 [OpenResty](openresty.html)。所有组件均可以方便的被激活或禁止。
+
+大部分组件默认是激活的，也有些也不是。
+
+Lua 5.1 版本中, [DrizzleNginxModule](drizzle-nginx-module.html)、[PostgresNginxModule](postgres-nginx-module.html)和[IconvNginxModule](iconv-nginx-module.html)默认并未开启。 你需要加入`--with-lua51`、`--with-http_drizzle_module`、`--with-http_postgres_module`和`--with-http_iconv_module` 编译选项来开启它们，其余各组件的编译选项，可对照[安装OpenResty](installation.html)说明，按需使能。
```

Comment:
> Lua 5.1 版本中

I'm afraid this is wrong. The ngx_drizzle and ngx_postgres modules have nothing to do with the standard Lua 5.1 interpreter.